pkg/kubelet/volumemanager/reconciler/reconstruct_common.go

```diff
@@ -172,7 +172,6 @@ func (rc *reconciler) cleanupMounts(volume podVolume) {
 	if err != nil {
 		metrics.ForceCleanedFailedVolumeOperationsErrorsTotal.Inc()
 		klog.ErrorS(err, mountedVolume.GenerateErrorDetailed("volumeHandler.UnmountVolumeHandler for UnmountVolume failed", err).Error())
-		return
```

Comment:
I'd leave it as it is. If some code will be added to the end of this function, absence of return can cause unexpected code behaviour. With the `return` the code is more explicit in my pov.